lightning/src/onion_message/messenger.rs

```diff
@@ -243,6 +243,51 @@ impl OnionMessageRecipient {
 	}
 }
 
+
+/// The `Responder` struct creates an appropriate [`ResponseInstruction`]
+/// for responding to a message.
+pub struct Responder {
+	/// The path along which a response can be sent.
+	reply_path: BlindedPath,
+	path_id: Option<[u8; 32]>
+}
+
+impl Responder {
+	/// Creates a new [`Responder`] instance with the provided reply path.
+	pub fn new(reply_path: BlindedPath, path_id: Option<[u8; 32]>) -> Self {
```

Comment:
We can make this private since `Responder` is supplied to the handler.

Reply:
Sounds good!
Addressed in d0b266c
Thanks!